src/content/docs/query-your-data/explore-query-data/dashboards/manage-your-dashboard.mdx

```diff
@@ -405,24 +405,27 @@ Exporting to PDF or as an image has certain limitations depending on the chart t
 ## Share your data [#data-share]
 
 You may be used to any user in your organization having the ability to create live chart URLs by using the <DoNotTranslate>**Get chart link**</DoNotTranslate> option in supported visualizations. Now, to prevent potential data exposures, administrators will have control over who can and can't create live chart URLs. This permission is now controlled by both the <DoNotTranslate>**Live Chart URL (self)**</DoNotTranslate> and <DoNotTranslate>**Live Chart URL (all)**</DoNotTranslate> capability, explained below. Using these, you can assign permissions to accounts to enable individual users to create shareable URLS.
-		
 
 In order to minimize disruption, customers who want to enable this functionality can create custom roles and assign them to their users prior to April 30, 2024. After April 30, 2024, this function will be generally available, and these capabilities will activate.
 
-**Note:** The ability to create “Live chart URLs” will be removed from <DoNotTranslate>HIPAA</DoNotTranslate> and <DoNotTranslate>Fedramp</DoNotTranslate> accounts.
+<Callout variant="important">
+The ability to create live chart URLs will be removed from <DoNotTranslate>HIPAA</DoNotTranslate> and <DoNotTranslate>Fedramp</DoNotTranslate> accounts.
```

Comment:
Probably out of scope, but this confused me. I thought there was a piece missing of "will be removed on EOL_DAY_HERE" - it took me a second to realize it meant the feature is locked for HIPAA and Fedramp accounts

Reply:
Agree, this confused me too. I'll ask Jeff to clarify this when he's back.